pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -445,7 +589,7 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 	//#endregion
 
 	//#region UPS DeploymentConfig
-	unifiedpushDeploymentConfig, err := newUnifiedPushServerDeploymentConfig(instance)
+	unifiedpushDeploymentConfig, err := newUnifiedPushServerDeployment(instance)
```

Comment:
I believe you said this function name difference was as a result of a rebase conflict. Should it be changed back?